lib/components/quiz_gridview.dart

```diff
@@ -1,10 +1,10 @@
 import 'package:flutter/material.dart';
 import 'package:maui/components/QuizButton.dart';
 
-class QuizGrid extends StatelessWidget {
+class QuizGridView extends StatelessWidget {
```

Comment:
Filename should be quiz_grid_view.dart or Classname should be QuizGrid.
CamelCase and snake_case should be equivalent